src/cljs/athens/events.cljs

```diff
@@ -383,55 +345,38 @@
 
 
 (defn select-down
-  [selected-items]
+  [selected-items next-block-uid]
   (let [editing-uid @(subscribe [:editing/uid])
         editing-idx (first (keep-indexed (fn [idx x]
                                            (when (= x editing-uid)
                                              idx))
                                          selected-items))
-        [_ f-embed]          (->> selected-items first db/uid-and-embed-id)
-        last-item            (last selected-items)
-        next-block-uid       (db/next-block-uid last-item true)]
+        [_ f-embed] (->> selected-items first db/uid-and-embed-id)]
     (cond
       (pos? editing-idx) (subvec selected-items 1)
-
       ;; shift down started from inside the embed should not go outside embed block
       f-embed            (let [sel-uid (str (-> next-block-uid db/uid-and-embed-id first) "-embed-" f-embed)]
                            (if (js/document.querySelector (str "#editable-uid-" sel-uid))
                              (conj selected-items sel-uid)
                              selected-items))
 
-      next-block-uid     (conj selected-items next-block-uid)
-      :else              selected-items)))
+      next-block-uid (conj selected-items next-block-uid)
+      :else          selected-items)))
 
 
 ;; using a set or a hash map, we would need a secondary editing/uid to maintain the head/tail position
 ;; this would let us know if the operation is additive or subtractive
 (reg-event-db
   :selected/down
   (fn [db [_ selected-items]]
-    (assoc-in db [:selection :items] (select-down selected-items))))
+    (let [last-item         (last selected-items)
+          next-block-uid    (db/next-block-uid last-item true)
+          ordered-selection (-> (into [] selected-items)
+                                (into [next-block-uid]))]
```

Comment:
Isn't this the same as `(into [next-block-uid] selected-items)`?

Reply:
It's more like `(into (into [] selected-items) [next-block-uid])`

Reply:
Yes I misread the threading operator, thought it was a thread last.